packages/frontend/web/components/MostViewed/MostViewedGrid.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useState } from 'react';
+import { css, cx } from 'emotion';
+import { headline, palette } from '@guardian/src-foundations';
+import { tablet, until } from '@guardian/pasteup/breakpoints';
```

Comment:
Ooh, can we use `src-foundations` for these? 🙏

Reply:
Good spot. Fixed.